content/guides/references/best-practices.md

```diff
@@ -515,10 +553,20 @@ assertions and don't worry about it
 
 We've seen many users writing this kind of code:
```

Comment:
I feel like the entire "Creating "tiny" tests with a single assertion" section is E2E-specific, and should get a badge saying so, and shouldn't have component testing examples.

Also, this line:

> Why you did this pattern in unit tests:

Should probably change to:

> Why you do this pattern in component and unit tests:

And this line:

> Why you shouldn't do this in Cypress:

Should probably change to:

> Why you shouldn't do this in End-to-End tests: